src/audio/aria/aria_hifi3.c

```diff
@@ -52,6 +52,7 @@ inline void aria_algo_calc_gain(struct aria_data *cd, size_t gain_idx,
 		in = audio_stream_wrap(source, in);
 		samples -= n;
 	}
+	max = max << 8;
```

Comment:
same s32_le format worry as for generic

Reply:
The concept of the fix has changed and this problem no longer exists.